src/strings-in-block/class-html.php

```diff
@@ -63,6 +65,7 @@ public function update( \WP_Block_Parser_Block $block, array $string_translation
 			$xpath      = new \DOMXPath( $dom );
 
 			foreach ( $block_queries as $query ) {
+				list( $query, $definedType ) = XPath::parse( $query );
```

Comment:
I think `$definedType` is not used here... maybe `list( $query )` only ?

Reply:
@strategio Done